metadata-service/factories/src/main/resources/application.yml

```diff
@@ -86,6 +86,9 @@ visualConfig:
   assets:
     logoUrl: ${REACT_APP_LOGO_URL:#{null}}
 
+trackingConfig:
```

Comment:
For a later time - we should think about bundling UI related configs (like the visualConfig above - into a single block since most of the other stuff is related to deeper backend config options. Will make it easier later if we have to split out FE from BE

Reply:
sounds good to me for sure!